### DISCUSSION TITLE
QoBuz

### DISCUSSION THREAD

**Discussion by X1andy** [BimmerGestalt/AAIdrive#644] [General]
Hi and compliments for your great  work. I use QoBuz on my home streamer and I start to use it also in my X1 with AAIdrive app. I see the icon and I can listen what I have start to listen on my mobile previously, but I can’t navigate on the app’s menu (playlists, search, favourites, archived, ecc.). Is possibile to resolve it? Thank you for your work and bye.
PS: I use also the app Radio.it for listen internet radio and it work perfectly..

**Comment by hufman** [comment]
I took a quick peek inside QoBuz last night and I think I [added support](https://github.com/BimmerGestalt/AAIdrive/commit/2a693abdedf3032983ad48647f6a36b41eac3495) to browse it, please try out the [latest build](https://bimmergestalt.s3.amazonaws.com/aaidrive/builds/androidautoidrive-latest-main-mapbox-sentry-release.apk) to see if it works!

**Comment by X1andy** [comment]
Hi Hufman. The new build seems to work, but I can’t try with the necessary attention now. Tomorrow I think I will check better. Thank you very much for your precious work.

**Reply by X1andy** [reply]
Hi Hufman. The new build work perfectly, now I can navigate in all the menus and in all the archives. There is only a particular  fact: all the music apps managed by AAIdrive, only at the first time, have to be started on the mobile. After this, all the following times, the icons appear on the car screen and I can use them  normally. I don’t remember if this fact is usual when I install a new build in clean mode, after deleting the previous. Really a good job.

**Reply by hufman** [reply]
I'm glad to hear it works!
I haven't heard of this oddity, maybe something about whether the music apps were fully discovered in the Phone UI after reinstalling?

**Reply by X1andy** [reply]
I have no idea about it. Maybe some authorisation missing at the music apps? It’s seems all regular. For sure isn’t a great trouble .
![5D06D6DF-FE12-43F5-9843-CDDC74FF2529](https://user-images.githubusercontent.com/120414334/207667966-5cd13906-232d-486d-bb4b-ca811164fe92.jpeg)

**Comment by X1andy** [comment]
hi, i have some bad news. Last night, after the car had been used by other family members, and some Hours of inactivity, when starting the car, the music app icons disappeared from the dashboard (except spotify), even though AAIdrive was regularly hooked. Again, only after launching the individual apps (see attached photo) did the icons appear on the dashboard. The strange thing is that using the stable version of AAIdrive downloaded from the Google play store, all the icons appear permanently, but obviously with that version QoBuz wasn't navigable (radio.it instead it was also in that version). Summary: with the modified build, the apps do not appear in the dashboard at start up but, once manually launched, they appear and work completely. But the procedure seems to have to be repeated after a period of inactivity of the car. 
![5EC37F0B-E9A5-493E-BB9C-04AAA378454A](https://user-images.githubusercontent.com/120414334/207800209-897522e0-0846-4d26-983d-6bd99e9ddceb.jpeg)

**Reply by hufman** [reply]
That screenshot shows that the music apps are only controllable while they are running. Try swiping down on that screen to rescan them.

**Reply by X1andy** [reply]
Wow, what surprize! Swiping down the screen to rescan the music apps, seems to resolve the problem. At every swip a new app changed status; now, after three  or four swips, the situation is as in the new screenshot. I haven’t the X1 to check on car now, but I will do it this afternoon and I will give you the feedback. I’m optimistic.
![0E0821B1-4756-48A4-BBF5-01CC2110BD2E](https://user-images.githubusercontent.com/120414334/208047458-232eb3b0-cecc-48da-a8d6-1a1fc2fd4c84.jpeg)

**Comment by X1andy** [comment]
Now all seems ok 👍 
Maybe a little bit more slow to connect, but all the apps work fine.

**Reply by hufman** [reply]
That's great to hear!
